infra/scripts/cometbft.toml

```diff
@@ -34,7 +34,7 @@ dependencies = ["cometbft-pull", "docker-network-create"]
 script = """
 id=`docker exec ${CMT_CONTAINER_NAME} cometbft show-node-id`
 ip=`docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ${CMT_CONTAINER_NAME}`
-echo $id@$ip:26656
+echo $id@$ip:${CMT_HOST_PORT}
```

Comment:
Is this correct? Looks like we changed from the P2P to the RPC port.

Reply:
I thought this was for something like the peer file, which enables containers to connect to each other without peer discovery, and 26656 was the port for that, which should work internally.

Reply:
Isn't this IP address only available inside the container?

Reply:
`26656` is the internal port, but if we want to run more than one CometBFT instance locally (which will definitely be the case if you are running a validator and a bootstrap), we need to use different ports for each of them. What we show here is the external port being exposed in the host to connect with this CometBFT instance. That way external peers have public visibility of this node, thus the `CMT_HOST_PORT`

Reply:
Okay, I was confused because in my smoke test script `CMT_HOST_PORT` is used to expose the RPC connection. Perhaps we can have a different name for this one, like `CMT_P2P_HOST_PORT` or `CMT_HOST_P2P_PORT`, and either keep this for RPC or add an variant for that with RPC in the name. 

The docs don't say, but if you intend these CometBFT nodes to be deployed on the Internet then you'd probably want the RPC node exposed too, otherwise nobody can send a transaction, at least not to this node.

Reply:
Fixed.